parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -908,7 +908,7 @@ resolve_packages_source_url() {
     PACKAGE_DOWNLOAD_BASE_URL="packages.aks.azure.com"
     for i in $(seq 1 $retries); do
       # Confirm that we can establish connectivity to packages.aks.azure.com before node provisioning starts
-      response_code=$(curl -s -o /dev/null -w "%{http_code}" --max-time 5 --noproxy "*" https://packages.aks.azure.com/acs-mirror/healthz)
+      response_code=$(curl -s -o /dev/null -w "%{http_code}" --max-time 5 https://packages.aks.azure.com/acs-mirror/healthz)
```

Comment:
This is called from `basePrep` right and HTTP Proxy setup is being done later on, will it work as expected in those environments?